TeamInfo.md

```diff
@@ -0,0 +1,6 @@
+Team 8 -
+Oyewale Oyediran	- waleoyediran@gmail.com
+Bharath Kumar	    - gilchristian.bharath@gmail.com
+Hanna Becker	    - hanna.e.becker@gmail.com
```

Comment:
Hannah is opting out of the group. Shall we take out her name?

Reply:
Yes we can remove her name. I will remove from my list. 
Can you add other member's name in your list too? I think this info has to be in everyone's repository. While submitting you should paste this list in the points to the reviewer.